events/wc_crisis_events.txt

```diff
@@ -0,0 +1,758 @@
+				home = 2209 # Fate's End
+				troops = {
+					archers = { 2000 2000 }
+					light_infantry = { 3000 3000 }
+					heavy_infantry = { 3750 3750 }
+					light_cavalry = { 500 500 }
+					knights = { 750 750 }
+				}
+				cannot_inherit = no
+				attrition = 1.0
+				reinforces = yes
+				maintenance = no
+			}
+			spawn_fleet = {
+				province = closest # closest sea zone
+				owner = event_target:target_gorak_tul
+				disband_on_peace = yes
+				troops = {
+					galleys = { 100 100 }
+				}
+			}
+			
+			create_character = {
+				name = Maloch
+				dynasty = 70202
+				culture = drust religion = throsic
+				attributes = {
+					martial=5 diplomacy=4 stewardship=5 intrigue=4 learning=4
+				}
+				trait=mastermind_theologian
+				trait=creature_vrykul
+				trait=class_necromancer_5
+				trait=zealous trait=cruel trait=proud trait=being_undead
+				age = 400
+			}
+			
+			new_character = {
+				set_graphical_culture = vrykul_undead
+			}
+			
+			create_character = {
+				name = Drif
+				dynasty = 70203
+				culture = drust religion = throsic
+				attributes = {
+					martial=6 diplomacy=3 stewardship=4 intrigue=5 learning=4
+				}
+				trait=brilliant_strategist
+				trait=creature_vrykul
+				trait=class_death_knight_5
+				trait=zealous trait=sturdy trait=brave trait=wroth trait=being_undead
+				age = 400
+			}
+			
+			new_character = {
+				set_graphical_culture = vrykul_undead
+			}
+			
+			create_character = {
+				name = Saor
+				dynasty = 70204
+				culture = drust religion = throsic
+				attributes = {
+					martial=5 diplomacy=5 stewardship=4 intrigue=4 learning=6
+				}
+				trait=grey_eminence
+				trait=creature_vrykul
+				trait=class_druid_5
+				trait=zealous trait=shrewd trait=cruel trait=temperate trait=patient trait=being_undead
+				age = 400
+				add_trait_undead_effect = yes
+			}
+			
+			new_character = {
+				set_graphical_culture = vrykul_undead
+			}
+		}
+		# Make the person that fired the decison a vassal of Gorak Tul
+		event_target:target_summoner_of_gorak = {
+			if = {
+				limit = {
+					character_can_be_vassalized_trigger = yes
+				}
+				set_defacto_liege = event_target:target_gorak_tul
+			}
+		}
+		# Declares war against Kul Tiras (wip)
+		k_kul_tiras = {
+			holder_scope = {
+				if = {
+					limit = {
+						dark_public_religion_trigger = no
+					}
+					reverse_unsafe_war = {
+                        target = event_target:target_gorak_tul
+                        casus_belli = invasion
+                        thirdparty_title = PREV
+                    }
+				}
+            }
+        }
+		
+		# The Thornspeakers take action
+		73 = { province_event = { id = WCCRS.149 days = 7 } }
+	}
+
+	option = {
+		name = EVTOPTA_WCCRS_130
+	}
+}
+
+# The Drust sack Boralus
+narrative_event = {
+	id = WCCRS.140
+	title = "EVTTITLE_WCCRS_140"
+	desc = {
+		trigger = { NOT = { religion = throsic } }
+		text = EVTDESC_WCCRS_140
+	}
+	desc = {
+		trigger = { religion = throsic }
```

Comment:
It makes more sense to check true religion. I shouldn't be happy if you're Holy Light in secret.
```suggestion
		trigger = { NOT = { true_religion = throsic } }
		text = EVTDESC_WCCRS_140
	}
	desc = {
		trigger = { true_religion = throsic }
```